docs/cli/server.md

```diff
@@ -96,7 +96,7 @@ The following options must be set to the same value on all servers in the cluste
 | --------------------------- | ------------------------ | -------------------------------------------------------------------------------------------- |
 | `--bind-address` value      | 0.0.0.0                  | k3s bind address                                                                             |
 | `--https-listen-port` value | 6443                     | HTTPS listen port                                                                            |
-| `--advertise-address` value | node-external-ip/node-ip | IPv4 address that apiserver uses to advertise to members of the cluster                      |
+| `--advertise-address` value | node-external-ip/node-ip | IPv4/6 address that apiserver uses to advertise to members of the cluster (note that the primary service cluster IP range should be of the same family of the advertised address) |
```

Comment:
We use `IPv4/IPv6` for other flag descriptions, and a nit on the wording.
I think the note will also look better on a separate line
```suggestion
| `--advertise-address` value | node-external-ip/node-ip | IPv4/IPv6 address that apiserver advertises for its service endpoint<br>Note that the primary `service-cidr` IP range must be of the same address family as the advertised address |
```